### PR TITLE
Rearrange bounding box

### PR DESCRIPTION
**Pull request by iory**
To rearrange and magnify bounding box

rearrange_bounding_box_publisher

### CODE REVIEW

File: jsk_pcl_ros/CMakeLists.txt [hunk 1]
```diff
@@ -176,6 +177,8 @@ jsk_pcl_nodelet(src/cluster_point_indices_decomposer_z_axis_nodelet.cpp
   "jsk_pcl/ClusterPointIndicesDecomposerZAxis" "cluster_point_indices_decomposer_z_axis")
 jsk_pcl_nodelet(src/resize_points_publisher_nodelet.cpp
   "jsk_pcl/ResizePointsPublisher" "resize_points_publisher")
+jsk_pcl_nodelet(src/rearrange_bounding_box_publisher_nodelet.cpp
+  "jsk_pcl/RearrangeBoundingBoxPublisher" "rearrange_bounding_box_publisher")
```
Comment (wkentaro) [inline jsk_pcl_ros/CMakeLists.txt]: Why not `rearrange_bounding_box`?

Comment (iory) [inline jsk_pcl_ros/CMakeLists.txt]: Refered to name of ResizePointsPublisher.

Comment (wkentaro) [inline jsk_pcl_ros/CMakeLists.txt]: > ResizePointsPublisher.

What a bad name..
I don't think you have to follow that name, but if you'd like to follow what about `rearranged_bounding_box_publisher`?

Comment (iory) [inline jsk_pcl_ros/CMakeLists.txt]: I'm not particular about it.
I should change it?

Comment (iory) [inline jsk_pcl_ros/CMakeLists.txt]: OK! I will change those names.

**Comment by iory** [comment]
Please merge this.

**Pull request merged**